src/Microsoft.AspNetCore.Server.Testing/Deployers/RemoteWindowsDeployer/RemoteWindowsDeployer.cs

```diff
@@ -53,6 +53,12 @@ public RemoteWindowsDeployer(RemoteWindowsDeploymentParameters deploymentParamet
                     " Account credentials are required to enable creating a powershell session to the remote server.");
             }
 
+            if (_deploymentParameters.ApplicationType == ApplicationType.Portable
+                && string.IsNullOrWhiteSpace(_deploymentParameters.DotnetRuntimePath))
+            {
+                throw new ArgumentException($"Invalid value '{_deploymentParameters.DotnetRuntimePath}' for {nameof(RemoteWindowsDeploymentParameters.DotnetRuntimePath)}. ");
```

Comment:
Ambiguous error message. Maybe something like "Invalid value for ... DotnetRuntimePath must be non-empty for portable apps."